product_attribute_value_menu/models/product_attribute_value.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 ForgeFlow, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductAttributeValue(models.Model):
+    _inherit = "product.attribute.value"
+
+    product_count = fields.Integer(string="Product", compute="_compute_product_count")
+
+    def _compute_product_count(self):
+        for value in self:
+            value.product_count = len(value.pav_attribute_line_ids)
+
+    def action_view_product(self):
+        action = self.env["ir.actions.act_window"].for_xml_id(
+            "product", "product_template_action"
+        )
```

Comment:
```suggestion
        action = self.env["ir.actions.act_window"]._for_xml_id(
            "product.product_template_action"
        )
```
When I try to open the products the action is failing due to this.